gapii/cc/call_observer.cpp

```diff
@@ -112,7 +113,7 @@ void CallObserver::invoke() {
 
 void CallObserver::encodeAndDeleteCommand(::google::protobuf::Message* cmd) {
     observePending();
-    auto encoder = mSpyPtr->getEncoder();
+    auto encoder = mSpyPtr->getEncoder(mApi);
```

Comment:
The Call Observer is re-created for every command. This is up to date with each command that may be run. (Vulkan/Gles/Core all set it when they create the call observer).